templates/mod/cache.conf.epp

```diff
@@ -0,0 +1,30 @@
+<% |
+  Optional[Array[String[1]]] $cache_ignore_headers    = undef,
+  Optional[Integer] $cache_default_expire             = undef,
+  Optional[Integer] $cache_max_expire                 = undef,
+  Optional[Apache::OnOff] $cache_ignore_no_lastmod    = undef,
+  Optional[Apache::OnOff] $cache_header               = undef,
+  Optional[Apache::OnOff] $cache_lock                 = undef,
+  Optional[Apache::OnOff] $cache_ignore_cache_control = undef,
+| -%>
+<%- if $cache_default_expire { -%>
+CacheDefaultExpire <%= $cache_default_expire %>
+<%- } -%>
+<%- if $cache_max_expire { -%>
+CacheMaxExpire <%= $cache_max_expire %>
+<%- } -%>
+<%- if $cache_ignore_no_lastmod { -%>
+CacheIgnoreNoLastMod <%= $cache_ignore_no_lastmod %>
+<%- } -%>
+<%- if $cache_header { -%>
+CacheHeader <%= $cache_header %>
+<%- } -%>
+<%- if $cache_lock { -%>
+CacheLock <%= $cache_lock %>
+<%- } -%>
+<%- if $cache_ignore_cache_control { -%>
+CacheIgnoreCacheControl <%= $cache_ignore_cache_control %>
+<%- } -%>
+<%- if $cache_ignore_headers { -%>
+CacheIgnoreHeaders <% $cache_ignore_headers.each |$header| { %><%= $header %> <% } %>
```

Comment:
In this case (list of words), prefer `join` which avoid a trailing space after the last item and is a bit more readable:

```suggestion
CacheIgnoreHeaders <% $cache_ignore_headers.join(' ') %>
```

Reply:
Nice. Added a missing '='.